layers/meta-resin-raspberrypi/recipes-bsp/bootfiles/rpi-config_git.bbappend

```diff
@@ -13,6 +13,8 @@ do_deploy_append() {
     echo "dtparam=audio=on" >> ${DEPLOYDIR}/bcm2835-bootfiles/config.txt
 }
 
-# On Raspberry Pi 3 serial console on ttyS0 is only usable if ENABLE_UART = 1
-# On builds set with DEBUG_IMAGE, we want serial console available on Raspberry Pi 3
-ENABLE_UART_raspberrypi3 = "${@bb.utils.contains('DISTRO_FEATURES','development-image','1','0',d)}"
+# On Raspberry Pi 3 and Raspberry Pi Zero WiFi serial console on ttyS0 is only
+# usable if ENABLE_UART = 1 On builds set with DEBUG_IMAGE, we want serial
```

Comment:
A missing period before "On".

Also DEBUG_IMAGE was renamed to DEVELOPMENT_IMAGE.